### PR TITLE
Ckeck characters words typos cypress test

### PR DESCRIPTION
**Pull request by AndersonCeci**
checks for Karaktere ,Fjale , Shenjimet 
It also presses SHIFT to make sure it doesnt count as a character as commented on the pull request #158 
I would like to not close this PR just yet and add the test for issue #159

### CODE REVIEW

**Comment by AndiBraimllari** [comment]
Utilize the `data-test` attribute instead of `id` for `#characterCountSpan`.

File: cypress/e2e/check-characters-words-typos.cy.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+describe("its going to test if the characters change, words, typos change as we write", () => {
```
Comment (AndiBraimllari) [inline cypress/e2e/check-characters-words-typos.cy.js:1]: Maybe we can reword this to "how `karaktere, fjalë, shenjime` change as we write"?

File: cypress/e2e/check-characters-words-typos.cy.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("will click on dokument and on shkruaj again", () => {
```
Comment (AndiBraimllari) [inline cypress/e2e/check-characters-words-typos.cy.js:6]: Ideally we break this test up into different tests, but if you want to keep it like this maybe we can reword it to "will correctly update `karaktere, fjalë, shenjime`"?

**Comment by AndersonCeci** [comment]
> Utilize the `data-test` attribute instead of `id` for `#characterCountSpan`.

Can you check it please I cant seem to make it work while testing it it doesn't grab the ID
